presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -552,6 +554,53 @@ void PrestoServer::populateMemAndCPUInfo() {
   **memoryInfo_.wlock() = std::move(memoryInfo);
 }
 
+void PrestoServer::runOperation(
+    proxygen::HTTPMessage* message,
+    proxygen::ResponseHandler* downstream) {
+  try {
+    ServerOperation op = buildServerOpFromHttpRequest(message);
+    switch (op.target) {
+      case ServerOperation::Target::kConnector:
+        http::sendOkResponse(downstream, connectorOperation(op, message));
+        break;
+    }
+  } catch (const VeloxUserError& ex) {
+    http::sendErrorResponse(downstream, ex.what());
+  } catch (const VeloxException& ex) {
```

Comment:
Why two catches?

Reply:
There is some non-user internal error that might be thrown. Just want to differentiate when exceptions got propagated.